Add installation and question usage guide (#20)
systems-cs-pub-ro/quiz-manager-moodle: doc/user_stories.md

> ## __User stories__:

Reviewer: ```suggestion
# User Stories

The table below is a summary of [user stories](https://www.atlassian.com/agile/project-management/user-stories) uses in the Quiz Manager Moodle plugin.

```

# User Stories

> The table below is a summary of [user stories](https://www.atlassian.com/agile/project-management/user-stories) uses in the Quiz Manager Moodle plugin.

Reviewer: ```suggestion
The table below is a summary of [user stories](https://www.atlassian.com/agile/project-management/user-stories) for the Quiz Manager Moodle plugin.
```
